event-app-api/server.js

```diff
@@ -94,14 +136,61 @@ app.get("/users", async (req, res) => {
   }
 });
 
-sequelize
-  .sync({ alter: true })
-  .then(() => {
-    const port = 3000;
-    app.listen(port, () => {
-      console.log(`App is listening on port ${port}`);
+//sending a get request to the events endpoint
+app.get("/events", async (req, res) => {
+  try {
+    const events = await Event.findAll({
+      // making the order of the events in a descending order depending on the time created
+      order: [["createdAt", "DESC"]],
     });
-  })
-  .catch((error) => {
-    console.error("Unable to connect to the database:", error);
+    res.json(events);
+  } catch (err) {
+    res.status(500).json({ message: err.message });
+  }
+});
+
+//logout button to delete the cookies once clicked
+app.get("/logout", async (req, res) => {
+  if (!req.cookies) {
+    res.status(401).end();
+    return;
+  }
+
+  const accessToken = req.cookies["access-token"];
+  if (!accessToken) {
+    res.status(401).end();
+    return;
+  }
+
+  res.cookie("access-token", "", {
+    httpOnly: true,
+    sameSite: "strict",
+    secure: true,
+    expires: new Date(),
   });
+  res.end();
+});
+
+//sending a POST request to the endpoint event_popup
+app.post("/event_popup", async (req, res) => {
+  const eventName = req.body.eventName;
+  const description = req.body.description;
+  const location = req.body.location;
+  const username = req.body.username;
+  const startTime = req.body.startTime;
+  const endTime = req.body.endTime;
+
+  //used the sequelize.query to inserting data into the events table
+  sequelize.query(
+    `INSERT INTO "public"."Events" (eventName, description, location , username, startTime, endTime, "createdAt", "updatedAt") VALUES ('${eventName}', '${description}', '${location}', '${username}', '${startTime}','${endTime}', CURRENT_TIMESTAMP, CURRENT_TIMESTAMP)`,
```

Comment:
What are the CURRENT_TIMESTAMP, CURRENT_TIMESTAMP for at the end? Is it creation time or update time?